Kajabi/sage-lib
# Sage icon font unicode fix

## Description
This PR corrects unicode issues with the previous Sage font update

## Test notes
1. Visit http://0.0.0.0:4000/pages/design/icon
2. Ensure all icons match to their name
